assets/configuration/configuration.json

```diff
@@ -0,0 +1,5 @@
+{
+  "editorLineNumbers": ["relative"],
```

Comment:
This isn't really related to this PR, but this should probably be `"editor.lineNumbers"` and `"editor.minimap.enabled"`, to mirror the VSCode equivalent. The tabline one doesn't have an equivalent I can think of, so less sure about that one.

Reply:
This is more about reading a `json` file to a record, I think renaming the keys is easy once this bit is done